lib/Utilities.ts

```diff
@@ -282,3 +282,40 @@ export function escapeDiscordMarkdown(text: string) {
         },
     );
 }
+
+export async function handleFloorPriceChannel(client: any)  {
```

Comment:
these don't really belong in Utilities imo it's not a utility function its performing, I would either put in index.ts or make a new file for periodically ran functions

Also regarding naming I would use `handle` for something that happens on an event or a user action, for example a `.on('click')`, or a `.on('message')` etc, this I would probably name something like `updateFloorPriceChannel`, however you don't need to modify it if you don't want.